dist/rust/riot-rust-linker-hook

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+[ -z "$BINDIR" ] && {
+    echo "$0: BINDIR not defined!" 1&>2
+    exit 1
+}
+
+_EXTRA_LIBS=${BINDIR}/_extra_libs
+mkdir -p ${_EXTRA_LIBS}
+
+while true; do
+    [ -z "$1" ] && exit 0
+    [ -f "$1" ] && {
+        if [[ "$1" == *.o ]]; then
```

Comment:
I want to match a pattern, that's why I'm using the bashism.

-----Original Message-----
From: Martine Lenders <notifications@github.com>
To: RIOT-OS/RIOT <RIOT@noreply.github.com>
Cc: Kaspar Schleiser <kaspar@schleiser.de>, Mention <mention@noreply.github.com>
Sent: Fr., 02 Sep. 2016 11:59
Subject: Re: [RIOT-OS/RIOT] initial Rust support (#5740)

> @@ -0,0 +1,21 @@
> +#!/bin/bash
> +
> +[ -z "$BINDIR" ] && {
> +    echo "$0: BINDIR not defined!" 1&>2
> +    exit 1
> +}
> +
> +_EXTRA_LIBS=${BINDIR}/_extra_libs
> +mkdir -p ${_EXTRA_LIBS}
> +
> +while true; do
> +    [ -z "$1" ] && exit 0
> +    [ -f "$1" ] && {
> +        if [[ "$1" == *.o ]]; then

@kaspar030 any reason why are not using single `[]`?

-- 
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub:
https://github.com/RIOT-OS/RIOT/pull/5740/files/bb0eedf0f0506f01cd968d64cec35b7478b18944#r77322526